src/uvw/util.hpp

```diff
@@ -11,6 +11,12 @@
 #include <array>
 #include <uv.h>
 
+#ifdef _WIN32
```

Comment:
I can't reproduce the error. The changes below compile fine. If you agree, I would apply them and merge this PR, then we can look over it and decide how to proceed. How did you reproduce the error? What's the environment?